framework/core/js/src/forum/ForumApplication.ts

```diff
@@ -126,7 +126,7 @@ export default class ForumApplication extends Application {
     // Route the home link back home when clicked. We do not want it to register
     // if the user is opening it in a new tab, however.
     document.getElementById('home-link')!.addEventListener('click', (e) => {
-      if (e.ctrlKey || e.metaKey || e.which === 2) return;
+      if (e.ctrlKey || e.metaKey || e.button === 1) return;
```

Comment:
Let's add a comment explaining this one

Reply:
are you referring to the conditional statement or just the `e.button` part?

Reply:
I think this is quite self-explanatory?

The comment above explains everything this line does.